src/twisted/conch/newsfragments/9765.feature

```diff
@@ -0,0 +1 @@
+twisted.conch.ssh now supports using RSA keys with SHA-2 signatures (RFC 8332).
```

Comment:
maybe also add a note that `rsa-sha2-512` and `rsa-sha2-256` are now automatically generated and set as the preferred host keys.

And in the release note, maybe add a note that the support is automatic and conch will use the existing RSA keys to generate SHA2 variants.

No API change is require for end users.

Reply:
I've extended this news fragment a bit to try to capture that.